### ISSUE TITLE
add stereo track combining error to errors list

### ISSUE THREAD

**Issue by LWinterberg** [audacity/audacity-support#37]
> I tested this on W10 with the latest master: audacity-win-3.4.0-alpha-20230609+0ec7ab2-x64-msvc2022
> 
> This works fine now:
> 
> 1. The creation of a stereo track is blocked
>    a) if one of the tracks has a Rate change from the TCP
>    b) if one of the tracks is Resampled with Tracks > Resample
> 2. Projects created in earlier Audacity versions with stereo tracks with mismatched channels (created either way) cannot be opened on 3.4.0 alpha.
> 
> @LWinterberg **_BUT_** the error message that is shown has a help link to the GitBook Audacity Support errors page, but there is nothing on that page to advise users as to how they can correct this situation.
> 
> ![image](https://user-images.githubusercontent.com/50205984/244862576-ee424d04-d92a-4885-a030-ce7177699e90.png)
> 
> Links to this page: https://support.audacityteam.org/troubleshooting/error-codes
> 
> And note that the error message in the image above does not actually have an "error code" as such _(but then several other errors dealt with on that page do not have errors codes either)_.

via @petersampsonaudacity in https://github.com/audacity/audacity/issues/4737#issuecomment-1585589077

**Comment by SteveDaulton** [comment]
Wouldn't it  be better for Audacity 3.4 to offer  to fix the problem (resample the lower sample rate channel to match  the other channel) rather than just telling the  user that their project is no longer supported?

**Comment by LWinterberg** [comment]
Yes, but we aren't sure if even a single user has this issue. If this error message gets meaningful traction in sentry it can be patched in later.
